### PR TITLE
#144 Predicate

### PR DESCRIPTION
**Pull request by VsSekorin**
As per https://github.com/yegor256/cactoos/issues/144

### CODE REVIEW

**Comment by 0crat** [comment]
@vssekorin I'm not managing this repo, remove the [webhook](https://github.com/yegor256/cactoos/settings/hooks) or contact me in [Slack](http://www.zerocracy.com) //cc @yegor256

**Comment by yegor256** [comment]
@VsSekorin this one doesn't make sense since it's exactly the same as `Func<T, Boolean>`. I think that we need it to return boolean and accept no arguments.

**Comment by VsSekorin** [comment]
@yegor256 no arguments. If I understand correctly, It will be logical value, not smth like java.util.function.Predicate?

**Comment by VsSekorin** [comment]
`[ERROR] /home/travis/build/yegor256/cactoos/src/test/java/org/cactoos/io/StickyInputTest.java:[46,33] incompatible types: cannot infer type-variable(s) T,X,Y`

**StickyInputTest** I don't know anything about this.

**Comment by VsSekorin** [comment]
**StickyInputTest** Because of this, the last PR were not build. I don't know anything about this class.

**Comment by yegor256** [comment]
@VsSekorin maybe `Term` would be a better name? https://en.wikipedia.org/wiki/Term_(logic)

**Comment by VsSekorin** [comment]
@yegor256 The term is a broader concept. Renamed to `LogTerm`.

File: src/main/java/org/cactoos/LogTerm.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+package org.cactoos;
+
+/**
+ * Logical value.
+ *
+ * <p>There is no thread-safety guarantee.
+ *
+ * @author Vseslav Sekorin (vssekorin@gmail.com)
+ * @version $Id$
+ * @since 0.5
```
Comment (yegor256) [inline src/main/java/org/cactoos/LogTerm.java]: @VsSekorin it's 0.7

**Comment by yegor256** [comment]
@VsSekorin and now it's exactly the same as `Scalar<Boolean>` :) I don't see why we need this new interface if it just duplicates what we already have. WDYT?

**Comment by VsSekorin** [comment]
@yegor256 my bad. The duplicate interface was deleted.

**Comment by VsSekorin** [comment]
@yegor256 I'm not sure, but it looks like the problem with Codecov is not in this PR.

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**